cms/static/js/views/content_group_edit.js

```diff
@@ -0,0 +1,40 @@
+define([
+    'js/views/list_item_edit', 'underscore'
+],
+function(ListItemEdit, _) {
```

Comment:
I keep seeing this problem (I don't think you introduced it) which is that the class name does not include View. The file name doesn't include ```_view``` because it is implied by the directory structure, but IMO that shouldn't mean that it has to be reflected in the variable name. I suppose it could be argued that an 'edit' is a type of view, but in that case I'd probably called this ListItemEditor.